src/migrations/20201205200637-create-grant.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     await queryInterface.createTable('Grants', {
       id: {
         allowNull: false,
-        autoIncrement: true,
+        autoIncrement: false,
```

Comment:
Same as the question for grantee, are we using HSES's id as our primary key?

Reply:
Yes, here too.